src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] list = new Figure [6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i <= 2; i++) {
+            list[i] = figureSupplier.getRandomFigure();
+            System.out.println(list[i].toString());

```

Comment:
Instead of calling `toString()` explicitly, you can simply pass the `list[i]` object to `System.out.println()` and it will call `toString()` implicitly.